### DISCUSSION TITLE
API barcode buddy items

### DISCUSSION THREAD

**Discussion by dennyreiter** [Forceu/barcodebuddy#242] [Q&A]
I first want to say this is awesome and has made getting starting with Grocy so much easier.
My SO is totally on board for scanning purchased and consumed items, but isn't going to spend the time creating unknown product in Grocy. Is there a way to query for new barcodes and unknown barcodes?  I don't think it would even need to be the actual items, perhaps just a count or even a binary sensor?  I'd like to have this pop up in Home Assistant notifying me I have items to take care.

Thanks again!

**Comment by Forceu** [comment]
That's a good idea, I will probably implement it with the next version. Tracked in #243
